### PR TITLE
Add several spelling corrections

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -14688,6 +14690,7 @@ fielname->filename
 fielneame->filename
 fiels->feels, fields, files, phials,
 fiercly->fiercely
+fifht->fifth
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:14693]:
```suggestion
fifht->fifth, fight,
```

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -20439,6 +20442,7 @@ missunderstood->misunderstood
 missuse->misuse
 missused->misused
 missusing->misusing
+mistakingly->mistakenly
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:20445]: Looks like it should be rare:
https://www.lexico.com/definition/mistakingly

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -23887,6 +23891,7 @@ possabilties->possibilities
 possabily->possibly
 possable->possible
 possably->possibly
+possbile->possible
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:23894]: Maybe also the phonetical(ish)
```suggestion
possbile->possible, possibly,
```

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -9357,6 +9357,7 @@ decrasing->decreasing, deceasing,
 decration->decoration
 decreace->decrease
 decreas->decrease
+decreate->decrease, discrete,
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:9360]: Maybe also the literal de-create
```suggestion
decreate->decrease, discrete, destroy,
```
Or the bad thing, but perhaps too unlikely, but maybe not much more than discrete:
```suggestion
decreate->decrease, discrete, desecrate,
```

**Review by peternewman — CHANGES_REQUESTED** [review]
A few comments

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: [c791666](https://github.com/codespell-project/codespell/pull/2410/commits/c7916664695cb623f756b4d86042f138b0e032f3)

File: codespell_lib/data/dictionary.txt (hunk 4 above)
Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: > Or the bad thing, but perhaps too unlikely, but maybe not much more than discrete:

Did you want to add desecrate too? You'd need to do some manual editing! :laughing:

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: Are you also suggesting `de-create` as a typo or solution?

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: @peternewman :point_up_2:

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: bump

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: @peternewman ?

**Review by skangas — APPROVED** [review]

File: codespell_lib/data/dictionary.txt (hunk 4 above)
Comment (DimitriPapadopoulos) [reply codespell_lib/data/dictionary.txt]: As a solution.

**Review by DimitriPapadopoulos — APPROVED** [review]

**Pull request merged**